client/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Используем Node.js официальный образ
```

Comment:
Nit: Please write comments in english. Up to you to fix it but it would be better to comment in english if you can